notebooks/book2/06/var_em_bound.ipynb

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        plt.xlabel("training time", fontweight="bold")
</pre><p>No need of bold I think</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>